<launch>
  <node name="main_state" pkg="main_state" type="main.py" output="screen" args="SpeechRecognition" />
  <include file="$(find pocketsphinx)/launch/RIPS.launch" />

</launch>
